<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToWeb</class>
 <widget class="QWidget" name="ExportToWeb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="etwUsername">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="etwPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="etwButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to Web</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>  password: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectFIle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Map Package</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectedMap">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Not Selected Yet</string>
   </property>
  </widget>
  <widget class="QLabel" name="response">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close</sender>
   <signal>clicked()</signal>
   <receiver>ExportToWeb</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
